prism-bukkit/src/main/resources/locale/messages-zh_CN.properties

```diff
@@ -113,25 +120,171 @@ prism.player-hover-no=否
 prism.player-hover-online=<gray>在线?
 prism.player-hover-uuid=<gray>UUID
 prism.player-hover-yes=是
-prism.prefix=<dark_gray>(<#ff55ff>！<dark_gray>) <#ff55ff>P<#f45aff>r<#e95fff>i<#dd64ff>s<#d269ff>m<gray> 》
-prism.preview-applied=<prefix><#9dfc56>预览已应用至世界
-prism.preview-cancelled=<prefix><#9dfc56>预览已取消, 所有方块已恢复至实时状态
-prism.purge-complete=<prefix><#9dfc56>清理完成! 总计删除: <#4fffd3><count><#9dfc56> 条记录
-prism.purge-cycle=<gray>》 定期清理删除了 <#4fffd3><result_count> <gray>条记录 (检查范围 <#4fffd3><result_lowerbound> <gray>- <#4fffd3><result_upperbound><gray>)
-prism.purge-starting=<prefix><#9dfc56>开始清理...
-prism.quantity=<quantity>
+prism.prefix=<dark_gray>(<\#ff55ff>！<dark_gray>) <\#ff55ff>P<\#f45aff>r<\#e95fff>i<\#dd64ff>s<\#d269ff>m<gray> 》
+prism.preview-applied=<prefix><\#9dfc56>预览已应用至世界
+prism.preview-cancelled=<prefix><\#9dfc56>预览已取消, 所有方块已恢复至实时状态
+prism.purge-complete=<prefix><\#9dfc56>清理完成\! 总计删除\: <\#4fffd3><count><\#9dfc56> 条记录
+prism.purge-cycle=<gray>》 定期清理删除了 <\#4fffd3><result_count> <gray>条记录 (检查范围 <\#4fffd3><result_lowerbound> <gray>- <\#4fffd3><result_upperbound><gray>)
+prism.purge-starting=<prefix><\#9dfc56>开始清理...
+prism.quantity=<quantity> 个
 prism.raid=袭击
-prism.reloaded-config=<prefix><#9dfc56>已重载配置
-prism.reloaded-locales=<prefix><#9dfc56>已重载语言文件
-prism.sign-minus=<#fc2150>-
-prism.sign-plus=<#9dfc56>+
-prism.teleporting-to-activity=<prefix> 正在传送至 <#03a5fc><activity_descriptor> <white><activity_action_past_tense> 由 <#03a5fc><activity_cause> <#4fffd3>@<yellow><activity_location>
+prism.reloaded-config=<prefix><\#9dfc56>已重载配置
+prism.reloaded-locales=<prefix><\#9dfc56>已重载语言文件
+prism.sign-minus=<\#fc2150>-
+prism.sign-plus=<\#9dfc56>+
 prism.teleporting-to=<prefix> 正在传送至 <grey><worldname> <x> <y> <z>
+prism.teleporting-to-activity=<prefix> 正在传送至 <\#03a5fc><activity_descriptor> <white><activity_action_past_tense> 由 <\#03a5fc><activity_cause> <\#4fffd3>@<yellow><activity_location>
+prism.time-ago=<time_ago> 前
 prism.unknown-cause=未知
-prism.vault-gui-title=<\#ff55ff>P<\#f45aff>r<\#e95fff>i<\#dd64ff>s<\#d269ff>m<gray> <bold><dark_aqua>查看
-prism.wand-activated=<prefix><#9dfc56>魔杖已激活, 模式: <wandmode>
-prism.wand-deactivated=<prefix><#9dfc56>已停用 <wandmode> 魔杖
+prism.vault-gui-next=<bold><dark_aqua>下一页
+prism.vault-gui-previous=<bold><dark_aqua>上一页
+prism.vault-gui-title=<\#ff55ff>P<\#f45aff>r<\#e95fff>i<\#dd64ff>s<\#d269ff>m<gray> <bold><dark_aqua>储物库
+prism.vault-header=<prefix><white>打开储物库，共有 <\#fcff69><results> <white>条结果
+prism.wand-activated=<prefix><\#9dfc56>魔杖已激活, 模式\: <wandmode>
+prism.wand-deactivated=<prefix><\#9dfc56>已停用 <wandmode> 魔杖
 prism.wand-mode-inspect=检查
 prism.wand-mode-restore=恢复
 prism.wand-mode-rollback=回滚
-prism.wand-switched=<prefix><#9dfc56>已切换魔杖模式至: <wandmode>
+prism.wand-switched=<prefix><\#9dfc56>已切换魔杖模式至\: <wandmode>
+rich.about=<prefix><white>版本 <\#4fffd3><version> <white>作者 <\#00f2fa>v<\#0af5d5>i<\#15f9b0>v<\#1ffc8b>e<\#29ff66>l<\#5cff77>e<\#8eff87>r<\#c1ff98>o<\#f3ffa8>i
```

Comment:
Everything was renamed to `prism.something` so the `rich.` and `text.` entries can all be removed, as they're duplicates now.